index/golioth.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "golioth",
     "description": "Golioth Cloud platform, which includes a device SDK, example code, and Reference Designs",
+    "avatar": "https://avatars.githubusercontent.com/u/58146410?s=48&v=4",
     "apps": [
         {
             "name": "reference-design-template",
+            "repo": "https://github.com/golioth/reference-design-template",
+            "docsUrl": "https://github.com/golioth/reference-design-template/blob/main/README.md",
             "title": "The Golioth Reference Design template",
             "description": "This starter project is a standalone application that includes databases (both stateful and time-based), Over-the-air update capabilities, settings service, remote procedure calls (RPCs)",
             "kind": "template",
             "tags": ["dfu","lte"],
+            "avatar": "https://avatars.githubusercontent.com/u/58146410?s=48&v=4",
```

Comment:
This is independent issue that I noticed now, these avatar URLs should not contain the search parameters, they should be added (stripped and replaced) by the component rendering them.

VSC-2913